### DISCUSSION TITLE
Giving a PIKU presentation at Scale21x in March '24

### DISCUSSION THREAD

**Discussion by jfmatth** [piku/piku#333] [General]
Hey everyone, 

I submitted a proposal for a presentation on PIKU for Scale21x and it was accepted :)

If you're in the Los Angeles area and can join us, please do.

For the PIKU team, any suggestions?  I've attached the PDF below

https://www.socallinuxexpo.org/scale/21x/presentations/i-just-want-push-my-code-git

Thanks,

J

**Comment by rcarmo** [comment]
Slides look great! Let us know what links we can add to the README once you’ve presented.

Happy Holidays!

**Reply by jfmatth** [reply]
Thanks @rcarmo :)

I'll let you know any feedback I get.

**Comment by mariusa** [comment]
I think Heroku should be mentioned as #1, for me Piku is a Heroku
replacement.

Marius

**Reply by jfmatth** [reply]
thanks @mariusa

which slide are you referring to?  Core values or something else?

**Comment by mariusa** [comment]
PaaS, which lists Google & Microsoft. These have various services, eg file
storage, database, firewall... That's why I personally think Heroku would
be a more appropriate comparison.

Marius

**Reply by jfmatth** [reply]
Thanks for that, i'll see what I can do to update it.  You're right that Heroku is closer to PIKU, but using the big boys to compare it to.

**Comment by jfmatth** [comment]
HI All, I've moved the PDF of the slides to the repo I'll be using as a demo (simple Django app).

I've also taken the suggestions from @mariusa about Heroku and incorporated them. 

Any feedback is appreciated.

The repo is here https://github.com/jfmatth/piku-django-scale
